samples/osgi-bundles/org.wso2.carbon.uuf.sample.simple-auth.bundle/pom.xml

```diff
@@ -42,10 +42,10 @@
             <groupId>org.wso2.carbon.messaging</groupId>
             <artifactId>org.wso2.carbon.messaging</artifactId>
         </dependency>
-        <dependency>
+        <!--dependency>
```

Comment:
Rather than commenting-out, if no longer used then let's remove this dependency.